src/main/java/com/jcraft/jsch/Session.java

```diff
@@ -129,6 +129,8 @@ public class Session {
 
   Buffer buf;
   Packet packet;
+  
+  List<String> fingerprintList = new ArrayList<String>();
```

Comment:
This can be shortened to simply `new ArrayList<>()` instead of `new ArrayList<String>()`.

Reply:
Changed it as suggested.